### PR TITLE
Remove unsupported features (Healthchecks, Directors and VCL settings) from `fastly_service_compute`

### PR DESCRIPTION
**Pull request by Integralist**
**BREAKING-CHANGE**: The features removed in this PR are only supported by `fastly_service_vcl` and although they might appear to have been successfully created for a `fastly_service_compute` resource, they actually were never supported and was a copy/paste mistake. Because we are now removing these features from the Terraform interface, it means we must consider this PR a breaking change that will result in a `2.0.0` release.

### CODE REVIEW

**Comment by Integralist** [comment]
**NOTE**: I've run the full acceptance test suite and it's passing.

File: docs/index.md [hunk 1]
```diff
@@ -113,4 +113,4 @@ The following arguments are supported in the `provider` block:
 - **api_key** (String) Fastly API Key from https://app.fastly.com/#account
 - **base_url** (String) Fastly API URL
 - **force_http2** (Boolean) Set this to `true` to disable HTTP/1.x fallback mechanism that the underlying Go library will attempt upon connection to `api.fastly.com:443` by default. This may slightly improve the provider's performance and reduce unnecessary TLS handshakes. Default: `false`
-- **no_auth** (Boolean) Set this to `true` if you only need data source that does not require authentication such as `fastly_ip_ranges`
+- **no_auth** (Boolean) Set this to `true` if you only need data source that does not require authentication such as `fastly_ip_ranges`
```
Comment (triblondon) [inline docs/index.md]: Should this be "a data source", or "data sources"?

File: docs/resources/service_compute.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: |-
 
 # fastly_service_compute
 
-Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/), which is run at the edge using [Lucet](https://github.com/bytecodealliance/lucet), an open-source WebAssembly runtime created by Fastly. A compute service encompasses Domains and Backends.
+Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/) using the `wasm32-wasi` target, which is run at the edge using [Wasmtime](https://github.com/bytecodealliance/wasmtime), an open-source WebAssembly runtime. A compute service encompasses Domains and Backends.
```
Comment (triblondon) [inline docs/resources/service_compute.md]:
```suggestion
Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/) using the `wasm32-wasi` target. A compute service encompasses Domains and Backends.
```

File: templates/resources/service_compute.md.tmpl [hunk 3]
```diff
@@ -8,7 +8,7 @@ description: |-
 
 # fastly_service_compute
 
-Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/), which is run at the edge using [Lucet](https://github.com/bytecodealliance/lucet), an open-source WebAssembly runtime created by Fastly. A compute service encompasses Domains and Backends.
+Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/) using the `wasm32-wasi` target, which is run at the edge using [Wasmtime](https://github.com/bytecodealliance/wasmtime), an open-source WebAssembly runtime. A compute service encompasses Domains and Backends.
```
Comment (triblondon) [inline templates/resources/service_compute.md.tmpl]:
```suggestion
Provides a Fastly Compute@Edge service. Compute@Edge is a computation platform capable of running custom binaries that you compile on your own systems and upload to Fastly. Security and portability is provided by compiling your code to [WebAssembly](https://webassembly.org/) using the `wasm32-wasi` target. A compute service encompasses Domains and Backends.
```

**Review by triblondon — APPROVED** [review]

**Pull request merged**